packages/components/src/index.tsx

```diff
@@ -0,0 +1,43 @@
+export { default as Advertisement } from './advertisement';
```

Comment:
needed to write all those exports manually because the glob-expand rollup plugin we used there doesn't work with typescript

Reply:
is there an issue at the rollup plugin maybe that we can track?

Reply:
Didn't find one. We would need to do a PR on their side

Reply:
ah, i see: https://github.com/gjbkz/rollup-plugin-glob-import/issues/364